rules/windows/execution_command_prompt_connecting_to_the_internet.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/02/18"
-integration = ["endpoint", "windows"]
+integration = ["endpoint", "windows", "sentinel_one_cloud_funnel"]
 maturity = "production"
-updated_date = "2025/03/20"
```

Comment:
this rule looks like it will require a dedicated tuning or refactoring in a separate PR (noise generator as-is)

Reply:
✍🏼  tracking

Reply:
@w0rk3r any issue for this to follow-up ?